spec/integration/puppet_x/puppetlabs/iis/powershell_manager_spec.rb

```diff
@@ -262,7 +241,7 @@ def close_stream(stream, style = :inprocess)
     end
 
     it "should return the exitcode of a script invoked with the call operator &" do
-      fixture_path = File.expand_path(File.dirname(__FILE__) + '../../../../exit-27.ps1')
+      fixture_path = File.expand_path(File.dirname(__FILE__) + '../../../../../exit-27.ps1')
```

Comment:
Looks like we're missing a `skip` from https://github.com/puppetlabs/puppetlabs-powershell/commit/11aedcc32158df693bf3894d1a72665ff87fe9d0